src/templates/_common/components/eu-blocked-modal.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const EuBlockedModal = () => (
+    <div id='eu-client-blocked-modal' className='invisible'>
+        <div id='eu-client-blocked'>
+            <div id='eu-client-blocked-header'>
+                <span>{it.L('Whoops!')}</span>
+            </div>
+            <div id='eu-client-blocked-message'>
+                {it.L('You cannot use your real money account with Deriv at this time.')}
```

Comment:
usually we use dialog to create a pop up instead of a new html template, but since the styling is very different this is fine. could you add a comment to explain why we're not using dialog?

Reply:
Sure thing, I went ahead with re-using the language modal as it looked similar to this one. Let me add a comment for clarity.